resources/js/remark-plugins/disable-constructs.ts

```diff
@@ -5,12 +5,40 @@ import type Constructs from 'micromark-core-commonmark';
 import type { Processor } from 'unified';
 import add from './add';
 
-export type DisabledType = 'chat' | 'default' | 'editor' | 'reviews';
+export type DisabledType = 'chat' | 'chatPlain' | 'default' | 'editor' | 'reviews';
 
 interface Options {
   type?: DisabledType;
 }
 
+function makeDisabledListFromAllowlist(allowlist: Construct[]) {
+  return ([
+    // 'characterEscape', // escaping things is always useful
+    // 'content', // not sure what this is
+
+    'attention',
+    'autolink',
+    'blankLine',
+    'blockQuote',
+    'characterReference',
+    'codeFenced',
+    'codeIndented',
+    'codeText',
+    'definition',
+    'hardBreakEscape',
+    'headingAtx',
+    'htmlFlow',
+    'htmlText',
+    'labelEnd',
+    'labelStartImage',
+    'labelStartLink',
+    'lineEnding',
+    'list',
+    'setextUnderline',
+    'thematicBreak',
+  ] as Construct[]).filter((item) => !allowlist.includes(item));
```

Comment:
should use a variable typed to `Construct[]` instead of `as Construct[]` as `as Construct[]` allows invalid types to go into the array